server/services/interventionsService.test.ts

```diff
@@ -1071,7 +1072,107 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
         },
       })
 
-      expect(await interventionsService.getInterventions(token)).toEqual([intervention, intervention])
+      expect(await interventionsService.getInterventions(token, {})).toEqual([intervention, intervention])
+    })
+
+    // TODO get feedback on how to stop these becoming tests of the backend logic - i.e. I want to test that it understands the filter that I'm asking it to apply, and I can't think of a way of verifying that other than asking it to filter something
```

Comment:
This is the open question I was referring to in the PR description - e.g. should the contract test contain things like:

- "not passing the `allowsMale` filter applies no restrictions on `allowsMale`"
- "passing `maximumAge` returns interventions whose `maximumAge` is precisely that value"

etc. And if so, how to test that?

Reply:
We spoke about this in the dev meeting just now, and concluded:

- we might have tests of the various filters, just to check that we're serializing params correctly (and we could possibly save even doing that if we were to use a POST body instead)
- we will not assert anything about the interpretation of the filters
- we will assert that a non-zero number interventions in the expected shape are returned, with a 200 status code